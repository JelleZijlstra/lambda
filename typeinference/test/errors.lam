#type bar a b = F int in
data foo a = F (bar a) | G int in
let f = \e. match e with {
	F x -> F x
	| G n -> G (n + 1)
} in
f
