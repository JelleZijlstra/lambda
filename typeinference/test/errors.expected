%%stdout%%
Error: Unbound type variable: bar
%%stderr%%
